android/src/com/frostwire/android/gui/SearchEngine.java

```diff
@@ -276,6 +277,14 @@ protected boolean isReady() {
         }
     };
 
+    public static final SearchEngine IDOPE = new SearchEngine("Idope", Constants.PREF_KEY_SEARCH_USE_IDOPE) {
+
+        @Override
+        public SearchPerformer getPerformer(long token, String keywords) {
+            return new IdopeSearchPerformer(token, keywords, DEFAULT_TIMEOUT / 2);
```

Comment:
why half a timeout?

Reply:
The page contain max 10 items. So it should be completed in short time. 🙄

Reply:
We need to consider that the app can be used from Smartphones in third world countries with slow connectivity.

Reply:
Also remember that FrostWire isn't connecting to that search engine only, it's also connecting to all the other search engines and it might take a while between a request and the handling of a response.

Reply:
Hmm. Thats true !